healpix_alchemy/point.py

```diff
@@ -29,20 +29,24 @@ def __init__(self, *args, ra=None, dec=None, **kwargs):
 
     x = Column(
         'x', Float, doc='Cached and precomputed Cartesian "x" coordinate',
-        **dfl(lambda ra, dec: cos(rad(ra)) * cos(rad(dec))))
+        **dfl(lambda ra, dec: cos(rad(ra)) * cos(rad(dec))
+              if ra is not None and dec is not None else None))
 
     y = Column(
         'y', Float, doc='Cached and precomputed Cartesian "y" coordinate',
-        **dfl(lambda ra, dec: sin(rad(ra)) * cos(rad(dec))))
+        **dfl(lambda ra, dec: sin(rad(ra)) * cos(rad(dec)
```

Comment:
Should be:

        **dfl(lambda ra, dec: sin(rad(ra)) * cos(rad(dec))
              if ra is not None and dec is not None else None))

Reply:
Good catch! Should be fixed now. Please take another look.